.azure-pipelines/common/setup.yml

```diff
@@ -0,0 +1,17 @@
+#
+# Azure Pipelines setup configuration for react on Linux, Windows, and macOS.
+#
+
+steps:
+  # master is not fetched by default and is needed to run prettier
+  - script: git fetch origin master:master
+    displayName: 'Checkout master'
```

Comment:
I wouldn't say "checkout" here since this implies the working directory is being switch over to master.

Reply:
Changed to "Fetch master branch"